modulemd/v2/modulemd-module-index.c

```diff
@@ -158,6 +153,32 @@ add_subdoc (ModulemdModuleIndex *self,
                        "Invalid mdversion for a stream object");
           return FALSE;
         }
+
+      if (stream == NULL)
+        return FALSE;
+
+      if (autogen_module_name &&
+          !modulemd_module_stream_get_module_name (stream))
+        {
+          name = g_strdup_printf ("__unknown_module_%d",
```

Comment:
Maybe instead change this to `__unnamed_module_%d` or `__nameless_module_%d`, to indicate it's actually just a module without a name?
This might be less confusing to people who are trying to figure out why their unnamed stream is invalid.

Reply:
Sure, I can paint the shed that color.